### ISSUE TITLE
Top bar still there, dont know how to remove

### ISSUE THREAD

**Issue by Logs22** [KiKaraage/ArcWTF#39]
Pretty much the title, I don't know how to get rid of this top bar. Other than that everthing seems to be working great! 
[
![image](https://github.com/KiKaraage/ArcWTF/assets/65024236/0ebb0e30-52d6-4080-a8da-c94b27cd9a64)
](url)

**Comment by KiKaraage** [comment]
Right-click on navbar > "Customize Toolbar" > uncheck "Title Bar" > press Done!
![gambar](https://github.com/KiKaraage/ArcWTF/assets/10529881/cec28aa8-9c7f-4762-8ece-06a5521c1018)

**Comment by harperreynell** [comment]
![screenshot_2024-03-28_14-19-41](https://github.com/KiKaraage/ArcWTF/assets/81856045/24fc94cf-f0a0-4a4c-8d41-5535ef7f6e24)
I don't have the "Title Bar" checkbox

**Comment by Kribbage** [comment]
There is an about:config setting called browser.tabs.inTitlebar , set it to true.
